File: MyHeart.DTO/Client/Client_QuestionAnswerDTO.cs
using MyHeart.DTO.ProfessionalInformation;
using System;
using System.Collections.Generic;
using System.Text;

namespace MyHeart.DTO
{
    public class Client_QuestionAnswerDTO : BaseEntityDTO
    { 
        public int SymptomQuestionId { get; set; }
        public SymptomQuestionsDTO SymptomQuestion { get; set; }
        public int UserId { get; set; }
        public DateTime CreatedDate { get; set; }
        public bool Approved { get; set; }
    }
}

File: MyHeart.Web/ClientApp/models/questionaire/ClientQuestionAnswerDTO.ts
import SymptomQuestionsDTO from "../professionInfo/SymptomQuestionsDto"

export default class ClientQuestionAnswerDTO {
    symptomQuestionId: number;
    symptomQuestion: SymptomQuestionsDTO
    userId: number;
    createdDate: Date;
    approved: boolean;
  }
  